src/cmd-build

```diff
@@ -275,7 +275,8 @@ build_image() {
 	# use NONE here instead of empty string because this has to survive through various string processing
 	ostree_remote="$(python3 -c 'import sys, yaml; print(yaml.safe_load(sys.stdin).get("ostree-remote", "NONE"))' < "$configdir/image.yaml")"
 	save_var_subdirs="$(python3 -c 'import sys, yaml; print(yaml.safe_load(sys.stdin).get("save-var-subdirs-for-selabel-workaround", "NONE"))' < "$configdir/image.yaml")"
-	kargs="$kargs console=tty0 console=${VM_TERMINAL},115200n8 ignition.platform.id=qemu"
+	[ "$arch" = "s390x" ] && tty="console=${VM_TERMINAL}" || tty="console=tty0 console=${VM_TERMINAL},115200n8"
```

Comment:
Why is this necessary? Is there harm to having `console=tty0 console=${VM_TERMINAL}` on s390x?

Reply:
As far as I know tty0 does not exist on s390x : https://github.com/systemd/systemd/blob/215bffe1b8d7cb72fe9f72ed53682d52d5c2a9c5/src/login/logind.c#L789. KVM on s390x uses ttysclp0 only.

I may need to ask s390x kernel dev about its tty but having `console=tty0` prevents showing some systemd logs (also openrc in Alpine Linux) in details. It should be okay keeping tty0 if you prefer, but we might need to remove it for development environment.

Reply:
Off topic, s390x automatically detects its tty, iirc we have never had the need to put `console=` for s390x in the past but since `VM_TERMINAL` was introduced recently, we went with the flow.
We can do something like `[ "$arch" != "s390x" ] && console=tty0 console=${VM_TERMINAL},115200n8`

Reply:
Huh, interesting. Lets do something like"
```
console=<standard console line>
# comment explaining lack of tty0 on s390x
[ "$arch" == s390x ] && console=<s390x console line>
```
Make it clear that s390x is the special case. and not involve the normal case with it's check

Reply:
Agreed.